manifests/0000_70_dns-operator_00-custom-resource-definition.yaml

```diff
@@ -1,20 +1,22 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: dnses.operator.openshift.io
+  creationTimestamp: null
+  name: dns.operator.openshift.io
 spec:
   group: operator.openshift.io
   names:
     kind: DNS
-    listKind: DNSList
-    plural: dnses
-    singular: dns
-  scope: Cluster
-  version: v1
-  subresources:
-    status: {}
+    plural: dns
+  scope: ""
```

Comment:
This looks totes wrong.

Reply:
This could be a problem: https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/crd/markers/crd.go#L39

Reply:
https://github.com/kubernetes-sigs/controller-tools/pull/220

Reply:
Will be fixed by https://github.com/openshift/api/pull/336 and https://github.com/kubernetes-sigs/controller-tools/pull/220